### DISCUSSION TITLE
Updated PPG II publication timeline

### DISCUSSION THREAD

**Discussion by joelnitta** [pteridogroup/ppg#91] [General]
I would like to update everyone on the progress of PPG, specifically with regards to the timeline for publication of the PPG II manuscript.

As [discussed](https://github.com/pteridogroup/ppg/discussions/89) at the PPG Workshop at IBC XX, we originally anticipated submitting the PPG II manuscript in Fall of 2024.

However, some things have changed since then. For one, we are prioritizing integration of PPG data into [World Flora Online](https://www.worldfloraonline.org/) (WFO) ahead of publication of PPG II. World Flora Online is a widely used taxonomic resource, so we want PPG II to be immediately available there when it is published. Furthermore, we will also be using the WFO taxonomic database manager, [Rhakhis](https://rhakhis.rbge.info/), to maintain PPG once it is integrated.

I (Joel Nitta) am currently working with the WFO staff to integrate PPG data into WFO. This is a lengthy process, since it involves reconciling all pteridophyte names currently in WFO to the new data. We aim to complete the initial integration by the time the next WFO data version is released, in December 2024. The version of the PPG data being provided comes directly from [Michael Hassler's World Ferns database](https://www.worldplants.de/world-ferns/ferns-and-lycophytes-list) (thank you Michael!). As such, it does not yet reflect all of the taxonomic decisions we have made as a group. My plan is to implement those changes using Rhakhis once the initial data integration into WFO is complete. And once that is done, we should be ready to move ahead with publication. So submission of the manuscript is likely to be sometime in 2025 (hopefully sooner rather than later!).

Please let me know if you have any questions.

Best Regards,

Joel

**Comment by schwartsburd** [comment]
Sounds good,
Pedro

**Reply by KMehltreter** [reply]
Dear Joel,
I think it makes sense to have the WFO updated before publishing PPG II. If WFO is updated Dec 2024, PPG II might be published in Spring or Summer 2025.
I think, we need a first deadline for the conclusion of the last proposals to be included. Is there one?
Regards,
Klaus

**Reply by joelnitta** [reply]
Thanks Klaus for pointing this out. We do not yet have such a deadline, but the steering committee will consider establishing one as you suggest.

On a related note, you may be aware of the [To-Do List](https://docs.google.com/spreadsheets/d/17pWet-jbCsKivPhl6FlDw9AheBcP5eKDxNahl7obUsw/edit?gid=0#gid=0) primarily maintained by @choess (Christopher Hoess). You can see from the list that we have voted on most of the new names published since PPG I. Of course systematic research is ongoing, and there will always be new names to consider, so I certainly agree with the need for a cutoff date for proposals ahead of PPG II.

**Comment by danieltejero54** [comment]
Dear Joel
There are many nomenclatural inconsistencies between WFO and world ferns. I trust the world ferns database more. I fully support the intervention in WFO 
Regards,
Daniel

**Reply by joelnitta** [reply]
WFO currently uses [Kew's World Checklist of Vascular Plants](https://powo.science.kew.org/about-wcvp) as its source of pteridophyte taxonomy, which is the reason for the inconsistencies with World Ferns and one of the reasons we want to prioritize integration of our data into WFO.

**Comment by GuguYanez** [comment]
Thanks for keeping us informed about the updates, Joel! The decisions and timeline you mentioned are fine with me. 

Please keep us in mind if you need help with any other step towards the preparation of the manuscript or the integration of data with WFO. Those of us who no longer have proposals to upload can also collaborate with the project in other ways, in addition to voting month by month of course :) :)
